File: Domain/Entities/WorkingHour.cs
using System;

namespace Domain.Entities
{
  public class WorkingHour
  {
    public int Id { get; set; }
    public int Day { get; set; }
    public TimeSpan OpenTime { get; set; }
    public TimeSpan CloseTime { get; set; }
    public int SportObjectId { get; set; }

    public virtual SportObject SportObject { get; set; }
  }
}

File: WebUI/client-app/src/app/models/workingHour.ts
export interface IWorkingHour {
  id: number;
  day: number;
  openTime: string;
  closeTime: string;
  sportObjectId: number;
}

export interface IWorkingHoursFormValues {
  sportObjectId: number;
  workingHours: IWorkingHour[];
}
